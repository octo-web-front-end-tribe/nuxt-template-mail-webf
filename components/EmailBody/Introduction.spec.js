import Vue from 'vue'
import Introduction from '@/components/EmailBody/Introduction'
import configuration from '@/contents/config';

jest.mock('@/contents/1_introduction.md', () => {
    return require('@/components/spec/MarkdownMock')
});

describe('Introduction Component', () => {
    let $mounted;

    const Constructor = Vue.extend(Introduction);

    beforeEach(() => {
        $mounted = new Constructor({
            propsData: {
               config: configuration
            }
        }).$mount()
    });

    test('snapshot', () => {
        let $html = $mounted.$el.outerHTML;
        expect($html).toMatchSnapshot()
    });
});