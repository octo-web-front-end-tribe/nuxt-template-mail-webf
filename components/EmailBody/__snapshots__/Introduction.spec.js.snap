// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Introduction Component snapshot 1`] = `
"<tr><td bgcolor=\\"#ffffff\\"><table role=\\"presentation\\" cellspacing=\\"0\\" cellpadding=\\"0\\" border=\\"0\\" width=\\"100%\\"><tr><td style=\\"padding: 20px 10px 20px; text-align: center;\\"><h1 style=\\"margin: 0px 0px 10px 0px; font-family: sans-serif; font-size: 30px; line-height: 125%; color: rgb(0, 24, 73); font-weight: bold;\\">
                        There is a better Web
                    </h1> <h2 style=\\"margin: 0px; font-family: sans-serif; font-size: 18px; font-style: italic; line-height: 125%; color: rgb(20, 160, 192); font-weight: normal;\\">
                        Nous accompagnons les acteurs du Web pour construire des architectures Front innovantes et de qualité.
                    </h2></td></tr> <tr><td style=\\"padding: 0px 10px 0px; font-family: sans-serif; font-size: 15px; line-height: 140%; color: rgb(85, 85, 85); text-align: left;\\"><p style=\\"margin: 0px;\\"><div>
    Je suis le fichier de mock pour les markdowns;
</div></p></td></tr> <tr style=\\"display: none;\\"><td style=\\"padding: 0px 10px 40px; font-family: sans-serif; font-size: 15px; line-height: 140%; color: rgb(85, 85, 85);\\"><table role=\\"presentation\\" cellspacing=\\"0\\" cellpadding=\\"0\\" border=\\"0\\" align=\\"center\\" style=\\"margin: auto;\\"><tr><td class=\\"button-td\\" style=\\"border-radius: 3px; background: rgb(34, 34, 34); text-align: center;\\"><a href=\\"https://google.com\\" target=\\"_blank\\" class=\\"button-a\\" style=\\"background: rgb(34, 34, 34); border: 15px solid #222222; font-family: sans-serif; font-size: 13px; line-height: 110%; text-align: center; text-decoration: none; display: block; border-radius: 3px; font-weight: bold;\\"><span class=\\"button-link\\" style=\\"color: rgb(255, 255, 255);\\">&nbsp;&nbsp;&nbsp;&nbsp;Découvre mes sources&nbsp;&nbsp;&nbsp;&nbsp;</span></a></td></tr></table></td></tr></table></td></tr>"
`;
